Client/src/components/common/PickDay/index.css

```diff
@@ -0,0 +1,13 @@
+.picker {
+  font-size: 14px;
+  font-family: Arial;
+  width: 180px;
+  color: darkblue;
+  height: 40px;
+  background-color: #d3d3d3;
+  border-radius: 5px;
```

Comment:
why you need it ??
if the border is none !!

Reply:
for the select list to be border not sharp 
![image](https://user-images.githubusercontent.com/42452940/56575254-f551f180-65cd-11e9-9461-4e75a684bfef.png)

Reply:
okay!  I got it